src/Uploadable/UploadableListener.php

```diff
@@ -260,7 +261,7 @@ public function processFile(AdapterInterface $ea, $object, $action)
 
         if ($config['allowedTypes'] || $config['disallowedTypes']) {
             $ok = $config['allowedTypes'] ? false : true;
-            $mimes = $config['allowedTypes'] ? $config['allowedTypes'] : $config['disallowedTypes'];
+            $mimes = $config['allowedTypes'] ?: $config['disallowedTypes'];
```

Comment:
I would like to check the possible values of `$config['allowedTypes']` and `$config['disallowedTypes']`, I think I can check this later.

Reply:
Yes, I agree. These cases deserve a new round:
https://github.com/doctrine-extensions/DoctrineExtensions/blob/bf8b828f1f68e0e0b6cb2a459051c08214285c8e/tests/Gedmo/Uploadable/Mapping/ValidatorTest.php#L221-L222